@xen-orchestra/lite/src/views/host/HostConsoleView.vue

```diff
@@ -102,7 +102,7 @@ const isConsoleAvailable = computed(() =>
   controlDomain.value !== undefined ? !isHostOperationPending(host.value!, STOP_OPERATIONS) : false
 )
 
-const consoleElement = ref()
+const consoleElement = useTemplateRef('consoleElement')
```

Comment:
To follow [vuejs](https://vuejs.org/guide/essentials/template-refs#accessing-the-refs) practice maybe you can you use kebab-case 
```suggestion
const consoleElement = useTemplateRef('console-element')
```